pkg/providers/v2/cloud.go

```diff
@@ -149,7 +396,7 @@ func (c *cloud) Routes() (cloudprovider.Routes, bool) {
 
 // HasClusterID returns true if the cluster has a clusterID
 func (c *cloud) HasClusterID() bool {
-	return false
+	return len(c.tagging.ClusterID) > 0
```

Comment:
I would be in favor of deferring tagging login in a follow-up PR. I _think_ we can work around any errors by setting the `--allow-untagged-cloud` flag.

Reply:
gotcha!